app/src/main/java/org/mozilla/fenix/search/SearchController.kt

```diff
@@ -70,8 +70,8 @@ class DefaultSearchController(
                 activity.startActivity(Intent(activity, CrashListActivity::class.java))
```

Comment:
We should be able to just delete this file and only use searchdialogcontroller going forward :)